lib/js/components/Drawer/DrawerTile/DrawerTile.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-	<ds-tile v-bind="$attrs" class="drawerTile" />
+	<div class="drawerTile">
+		<ds-tile v-bind="$attrs" />
+	</div>
```

Comment:
It emits `click` event even if disabled. It's the same on master but maybe it's a good opportunity to fix it?

This is ignored:
https://github.com/bethinkpl/design-system/blob/33ebab65073bc127bb8b0fc516e4c42ce864fbe9/lib/js/components/Tile/Tile.vue#L171

Reply:
https://github.com/bethinkpl/design-system/pull/310/commits/f68b23b668af14803d0be0448a4bdc62f4dae94a